spec/services/tax_tribs/mapping_code_determiner_spec.rb

```diff
@@ -31,12 +31,14 @@ def closure_case_type; end
 
   context 'when there is no case_type nor closure_case_type' do
     it { is_expected.to fail_to_determine_mapping_code }
+    it { expect(subject.valid_for_determining_mapping_code?).to eq(false) }
   end
 
   context 'when there is a penalty and it is level 1' do
     let(:penalty_level) { PenaltyLevel::PENALTY_LEVEL_1 }
 
     it { is_expected.to have_mapping_code(:appeal_penalty_low) }
```

Comment:
I think I will leave this matcher, IMO it makes the tests more readable, and it is playing nice with Mutant at the moment. But will remove the other matcher.